src/preact/hooks/useLocalStorage.js

```diff
@@ -0,0 +1,29 @@
+import {useCallback, useRef, useState} from '#preact';
+
+// eslint-disable-next-line local/no-forbidden-terms, local/no-global
+const storage = window.localStorage;
```

Comment:
Extracting `localStrage` in module scope prevents this entire module from being used in SSR. Please move the property access into the two functions.

Reply:
Done